static/canvas.js

```diff
@@ -23,7 +23,7 @@ class Canvas {
         this.selectionColor = '#CC0000';                   // Border color of selected boxes.
         this.selectionWidth = 2;                           // Border fillSize of selected boxes.
         this.selectionCorner = 5;                          // Corner fillSize of selected boxes.
-        this.interval = 30;                                // Frequency to be redrawn.
+        this.interval = 40;                                // Frequency to be redrawn.
```

Comment:
This was because too much computation was happening during 30ms? If so, what if the user has a slow computer?